content/blog/2021-11-05.md

```diff
@@ -0,0 +1,206 @@
+
+- The XSF offers [fiscal hosting](https://xmpp.org/community/fiscalhost/) for XMPP projects now! Please apply via Open Collective. For more information, see the [announcement blog post](https://xmpp.org/2021/09/the-xsf-as-a-fiscal-host/).
+
+    - Moreover, the XSF has accepted its first project to the fiscal hosting program! A big welcome to the [MAM Plugin for XMPP.js](https://gitlab.com/deblounge/xmppjs-mam-plugin) (note that this is unaffiliated with the upstream [xmpp.js](https://github.com/xmppjs/xmpp.js) project)! They will be working on adding MAM support to xmpp.js with the eventual goal of adding it to the Matrix Bifrost bridge, allowing XMPP users to fetch history from their favorite Matrix channels. You can [donate to the effort here](https://opencollective.com/mam-plugin-for-xmppjs).
+
+- The XSF is planning to participate the Google Summer of Code 2022 (GSoC). If you are interested in participating as a student, mentor or as project in general please [add your ideas and reach out to us](https://wiki.xmpp.org/web/Google_Summer_of_Code_2022)!
+
+- The blog and newsletter XSF pages offer now multiple languages. We are happy for volunteers to support translating!
+
+## Events
+
+[XMPP Office Hours](https://wiki.xmpp.org/web/XMPP_Office_Hours) - Also, checkout our new [YouTube channel](https://www.youtube.com/channel/UCf3Kq2ElJDFQhYDdjn18RuA)!
+
+[Berlin XMPP Meetup (remote)](https://mov.im/?node/pubsub.movim.eu/berlin-xmpp-meetup): Monthly Meeting of XMPP Enthusiasts in Berlin - always 2nd Wednesday of the month.
+
+## Videos
+
+Sam Whited gave a [talk for the XMPP Office Hours](https://www.youtube.com/watch?v=lprIwxyPY2E) about the XSF's new fiscal hosting service!
+
+TBA: Monal presentionation at Berlin XMPP Meetup: LINK
+Guest talk at Berlin XMPP Meetup: [Diving deep into Briar at the XMPP Meetup Berlin](https://nico.dorfbrunnen.eu/posts/2021/diving-at-xmpp/)
+
+## Articles
+
+After being featured in the [September 2021](https://xmpp.org/2021/10/the-xmpp-newsletter-september-2021/) newletter with a Movim & Element web client comparison, [Ade Malsasa Akbar](https://floss.social/@ademalsasa) is back with [a short tutorial](https://www.ubuntubuzz.com/2021/10/how-to-voice-video-calls-on-xmpp-and-matrix-made-simple.html) on how to use Audio and Video calls in these clients.
+
+Have you heard about [Bad XMPP](https://badxmpp.eu/)? Well, it is actually good - you can test your clients and servers against a set of intentionally badly configured XMPP services. Happy debugging!
+
+[INSERT IMAGE]
+
+Maybe use the top as picture to better advertise the project (+1 for using the Website's header including BadXMPP's logo)
```

Comment:
TODO reminder on this and the previous image so that it doesn't get forgotten about and merged this way :)

Reply:
1c7bf93